README.md

```diff
@@ -21,51 +21,60 @@ To view it, open `html/index.html` in your web browser.
 You can simulate running a web server using Python, which will automatically
 open `index.html` when following a link to a directory:
 
-    $ cd html
-    $ python3 -m http.server -b localhost 8000 &
+    cd html ; python3 -m http.server -b localhost 8000 &
 
 ## Contributing to the Screenshots Section
 
 To contribute with Screenshots:
 
 1. Add your image to the folder images/screenshots with an appropriate name.
 
-2. Add a new `<figure>` tag in the bottom of *screenshots.mdwn*,
+1. Add a new `<figure>` tag in the bottom of *screenshots.mdwn*,
 
-3. Inside the new tag, add the screenshots with a <img> tag and use the `<figcaption>` to add caption to the image, explaining what is being used on the screenshot.
+1. Inside the new tag, add the screenshots with a `<img>` tag and use the
+   `<figcaption>` to add caption to the image, explaining what is being used on
+   the screenshot.
```

Comment:
There is a setting for this - I personally like the ordered style better.

Reply:
Something along this maybe?  (had this laying around from the last time)

```
% cat .markdownlint-style.rb
all

rule 'MD029', :style => "ordered"

% cat .mdlrc
style ".markdownlint-style.rb"
```

Reply:
I briefly looked at it, saw that it required adding another file to the repo and thought "no!". :-/